cedar-policy-core/src/ast/policy.rs

```diff
@@ -279,9 +285,9 @@ fn describe_arity_error(unbound_values: &[SlotId], extra_values: &[SlotId]) -> S
         // PANIC SAFETY 0,0 case is not an error
         #[allow(clippy::unreachable)]
         (0,0) => unreachable!(),
-        (_unbound, 0) => format!("The following slots were unbound: {}", unbound_values.iter().join(",")),
-        (0, _extra) => format!("The following slots were provided as arguments, but did not exist in the template: {}", extra_values.iter().join(",")),
-        (_unbound, _extra) => format!("The following slots were unbound: {}\nThe following slots were provided as arguments, but did not exist in the template: {}", unbound_values.iter().join(","), extra_values.iter().join(","))
+        (_unbound, 0) => format!("the following slots were not provided as arguments: {}", unbound_values.iter().join(",")),
+        (0, _extra) => format!("the following slots were provided as arguments, but did not exist in the template: {}", extra_values.iter().join(",")),
+        (_unbound, _extra) => format!("the following slots were not provided as arguments: {}\nthe following slots were provided as arguments, but did not exist in the template: {}", unbound_values.iter().join(","), extra_values.iter().join(","))
```

Comment:
meta comment we can sort out outside this PR: How do we feel about new lines in error text?

Reply:
they're probably not good in the long run.  Make an issue?

Reply:
Newlines in errors are great when they improve formatting and are properly indented and provide context and etc, which this here is not necessarily doing.  Once we have `miette`, which does the sophisticated newlines and indentation and formatting, I suspect a message like this with embedded newlines will break things